### DISCUSSION TITLE
Proxy: Path blocking & CSS Assets

### DISCUSSION THREAD

**Discussion by vibarco** [pglombardo/PasswordPusher#1373] [Q&A] [answered]
Hello,

When clicked on the url to retrieve passwords, no css is loading. Attached.

Best!

![no-css](https://github.com/pglombardo/PasswordPusher/assets/33134255/7e5d76ee-cb9b-438e-84b4-d4056f9557eb)

**Comment by vibarco** [comment]
It seems that i am blocking css /assets/application-4811f9a7a475b60ae3c5070af5928801ee5707f0f8caff8d03bea292d17da0f3.css, with a too restrictive mathing path_regexp:

`@mypath path_regexp ^/.*/p/.*`

I am just trying to block anything that not cointains /p/ in the url to only allow access to the ephemeral link from the outside.

With no luck. Any ideas?

Best

**Reply by pglombardo** [reply, marked as answer]
The easiest way to achieve this is to simply use `PWP__ALLOW_ANONYMOUS=false` which will require people to log in to access the front page. Secret URLs will still be public. This feature requires logins to be enabled.

All of the settings are laid out in the [Configuration documentation](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md).

If that's not a solution for you, some other ideas:

1. Did you set a custom theme?  Make sure to also set `PWP_PRECOMPILE=true` so that assets are recompiled
2. See also [this discussion](https://github.com/pglombardo/PasswordPusher/discussions/1364) for potentially a few more leads on proxy configuration

**Reply by vibarco** [reply]
Thank you Peter, solved.

**Reply by troubleshootme** [reply]
@vibarco What unauthenticated path did you add to get the css and js to load properly?

**Reply by troubleshootme** [reply]
I got it working by adding:
```
/*/p/*
/p/*
/*/f/*
/f/*
/*/r/*
 /r/*
^/assets/*.*
```

**Reply by vibarco** [reply]
Many thanks @troubleshootme solved! Finally i try this:

```
 @allow_path {
    path /en/p/*
    path /assets/*.*
  }
```
Best

**Comment by Patosb500** [comment]
> It seems that i am blocking css /assets/application-4811f9a7a475b60ae3c5070af5928801ee5707f0f8caff8d03bea292d17da0f3.css, with a too restrictive mathing path_regexp:
> 
> 
> 
> `@mypath path_regexp ^/.*/p/.*`
> 
> 
> 
> I am just trying to block anything that not cointains /p/ in the url to only allow access to the ephemeral link from the outside.
> 
> 
> 
> With no luck. Any ideas?
> 
> 
> 
> Best
